src/pages/socialGroups/SocialGroups.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import { useIntl } from 'react-intl';
+
+import AddIcon from '@material-ui/icons/Add';
+
+import useSocialGroups from '../../models/socialGroups/useSocialGroups';
+import useDeleteSocialGroup from '../../models/socialGroups/useDeleteSocialGroup';
+import useDocumentTitle from '../../hooks/useDocumentTitle';
+import { cellRendererTypes } from '../../components/dataDisplays/cellRenderers';
+import MainColumn from '../../components/MainColumn';
+import LoadingScreen from '../../components/LoadingScreen';
+import SadScreen from '../../components/SadScreen';
+import Text from '../../components/Text';
+import ActionIcon from '../../components/ActionIcon';
+import Button from '../../components/Button';
+import ConfirmDelete from '../../components/ConfirmDelete';
+import SettingsBreadcrumbs from '../../components/SettingsBreadcrumbs';
+import DataDisplay from '../../components/dataDisplays/DataDisplay';
+import AddSocialGroupDialog from './AddSocialGroupDialog';
+
+export default function SocialGroups() {
+  const intl = useIntl();
+
+  const { data, loading, error, statusCode } = useSocialGroups();
+  const {
+    mutate: deleteSocialGroup,
+    loading: deletePending,
+    error: deleteError,
+    clearError: clearDeleteError,
+  } = useDeleteSocialGroup();
+
+  useDocumentTitle('SOCIAL_GROUPS');
+  const [addDialogOpen, setAddDialogOpen] = useState(false);
+  const [pendingDeleteGuid, setPendingDeleteGuid] = useState(null);
+
+  const columns = [
+    {
+      id: 'name',
+      name: 'name',
+      labelId: 'NAME',
+      options: {
+        cellRenderer: cellRendererTypes.capitalizedString,
+      },
+    },
+    {
+      id: 'guid',
+      name: 'actions',
+      labelId: 'ACTIONS',
+      options: {
+        customBodyRender: (_, { guid }) => [
```

Comment:
Zuh? What does it mean if the second argument is an object with a provided property?... I remember that the second argument was the whole object, but I've never seen this before.... is it just kinda auto-extracting the guid property?

Reply:
yes that's exactly what it does!